scripts/wsrep_sst_common.sh

```diff
@@ -269,13 +269,13 @@ parse_cnf()
     fi
 
     # look in group
-    if [ -z $reval ]; then
+    if [ -z "$reval" ]; then
```

Comment:
Is this commit based on some old mariadb version , This does not exist in latest mariadb (not in 10.0-galera , 10.1 , 10.2 ,10.3)

Reply:
I'm not sure how you can say it doesn't exist as it is right there in master.. The issue was produced with an image based off of https://hub.docker.com/_/mariadb/ (mariadb:10.1). However, it looks like there is already a broader effort underway to fix these errors in #581.

Reply:
The 10.3 got changed between when cloned/pulled the branch and got submitted. Not a big deal. The second chunk of this applies with some fuzz.

A compressor/decompressor with args the test suite would be good to detect these forms of issues. Even ifs its something more common like gzip (less build/test dependencies for distros)

Reply:
Right, thanks , Working on compressor/decompressor test